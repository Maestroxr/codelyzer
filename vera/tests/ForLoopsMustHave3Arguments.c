
int main()
{

	
	int x = 0, i = 0;
	
	/*OK*/
	for (i = 0; i < 10; i++) 
	{
		x++;
	}
	
	for (int j = 0; j < 10; i++) 
	{
		j++;
	}
	/*--------*/
	
	/*PROBLEM*/
	for (i = 0; i < 10; 4) 
	{
		x++;
	}
	for (i = 0; 5 < 10; i++) 
	{
		x++;
	}
	for (; i < 10; i++)
	{
		x++;
	}
	for (i = 0; i < 10; )
	{
		x++;
	}
	/*********/
	
	
	return x;
}

