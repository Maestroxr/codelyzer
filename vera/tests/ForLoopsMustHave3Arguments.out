C:\Work\AutoAssessment\codelyzer\vera\tests\ForLoopsMustHave3Arguments.c:21: For loops must have all three loop arguments present.
C:\Work\AutoAssessment\codelyzer\vera\tests\ForLoopsMustHave3Arguments.c:25: For loops must have all three loop arguments present.
C:\Work\AutoAssessment\codelyzer\vera\tests\ForLoopsMustHave3Arguments.c:29: For loops must have all three loop arguments present.
C:\Work\AutoAssessment\codelyzer\vera\tests\ForLoopsMustHave3Arguments.c:33: For loops must have all three loop arguments present.
